### PR TITLE
allow math operations in madcow.

### PR DESCRIPTION
**Pull request by framos4impact**

### CODE REVIEW

File: madcow-webdriver/src/main/groovy/au/com/ps4impact/madcow/runner/webdriver/blade/EvalMath.groovy [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                builder.append("$v ")
+            }else{
+                builder.append("$it ")
+            }
+        }
+        return builder.toString()
+    }
+
+    //checks if the entered variable is valid within string
+    public static boolean isValidJavaIdentifier(String s) {
```
Comment (patsimon) [inline madcow-webdriver/src/main/groovy/au/com/ps4impact/madcow/runner/webdriver/blade/EvalMath.groovy]: Does this need to be public and static?

File: madcow-webdriver/src/main/groovy/au/com/ps4impact/madcow/runner/webdriver/blade/EvalMath.groovy [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        if (s.isEmpty()) {
+            return false;
+        }
+        if (!Character.isJavaIdentifierStart(s.charAt(0))) {
+            return false;
+        }
+        for (int i = 1; i < s.length(); i++) {
```
Comment (patsimon) [inline madcow-webdriver/src/main/groovy/au/com/ps4impact/madcow/runner/webdriver/blade/EvalMath.groovy]: refactor to ```return s.toCharArray().every { Character.isJavaIdentifierPart(it) }```

File: madcow-webdriver/src/test/groovy/au/com/ps4impact/madcow/runner/webdriver/blade/EvalMathTest.groovy [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    protected verifyExecution(GrassBlade blade, boolean shouldPass, String resultingMessage = null) {
+        (testCase.stepRunner as WebDriverStepRunner).driver.get("file://${testHtmlFilePath}");
+        MadcowStep step = new MadcowStep(testCase, blade, null);
+        testCase.stepRunner.execute(step);
+        String sum = step.testCase.grassParser.getDataParameter("@sum")
+        assertEquals("3.00",sum)
+    }
+
+    @Test
+    void testEvalMath() {
```
Comment (patsimon) [inline madcow-webdriver/src/test/groovy/au/com/ps4impact/madcow/runner/webdriver/blade/EvalMathTest.groovy]: Can we please get another test case for the negative case?
